src/routes/channel.rs

```diff
@@ -148,16 +148,23 @@ pub async fn get_summary(state: State<AppState>, Path(channel): Path<String>) ->
 
     match initial_content {
         Some(content) => {
-            // make summary a default json object
+            // Make summary a default JSON object
             let mut summary = json!({});
 
-            // add items to summary
-            summary["items"] = json!(content.items.len());
+            // Get items from content across all versions
+            let items = match content {
```

Comment:
nit: not necessary, but you could add a `items()` function to the `ChannelContent` enum to hide this match.

e.g. in `model.rs`, add:
```rust
impl ChannelContent {
    pub fn items(&self) -> Vec<Item> {
        match self {
            ChannelContent::ChannelContentV1 { items, .. } => items,
            ChannelContent::ChannelContentV2 { items, .. } => items,
        }
    }
}
```

then just call `content.items()` here and in the `validate_channel_content` function below.